core/drand_test.go

```diff
@@ -570,3 +575,110 @@ func CloseAllDrands(drands []*Drand) {
 		//os.RemoveAll(drands[i].opts.dbFolder)
 	}
 }
+
+func TestDrandDKGFreshWithExecutableEntropy(t *testing.T) {
+	n := 5
+	nbRound := 2
+	period := 1000 * time.Millisecond
+
+	file, err := os.Create("./veryrandom.sh")
```

Comment:
let's create that in a tmp folder using https://golang.org/pkg/io/ioutil/#TempFile